cdk.json

```diff
@@ -27,6 +27,7 @@
         "region": "ap-southeast-2",
         "vpc_id": "vpc-23487b47",
         "subnets": ["subnet-51844336", "subnet-a6a85fef", "subnet-98f2a8c1"],
+        "rds_fdw_instance_type": { "class": "BURSTABLE3", "size": "SMALL" },
```

Comment:
There's `BURSTABLE3` and `SMALL` constants in the `aws_ec2` library; we should probably use those.